include/Zydis/DecoderTypes.h

```diff
@@ -149,14 +149,16 @@ typedef struct ZydisDecodedOperandMem_
      */
     struct ZydisDecodedOperandMemDisp_
     {
-        /**
-         * Signals, if the displacement value is used.
-         */
-        ZyanBool has_displacement;
+
         /**
          * The displacement value
          */
         ZyanI64 value;
+        
+        /**
+        * The size of the displacement value
```

Comment:
Misaligned comment. Also there are extra blank lines before/after structure fields (inconsistent with rest of the header).